execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -256,12 +251,14 @@ impl LmdbWasmTestBuilder {
                 .expect("should create LmdbTrieStore"),
         );
 
-        let global_state =
-            LmdbGlobalState::empty(environment, trie_store).expect("should create LmdbGlobalState");
+        let max_query_depth = DEFAULT_MAX_QUERY_DEPTH;
```

Comment:
Nit: Could we maybe get this value similar to how we derive the `PRODUCTION_GENESIS_REQUEST` to minimize drift between chainspec and default values

Reply:
maybe...ill look at it in the upcoming PR that insinuates the data provider into the test builder.